client/components/mma/switch/SwitchOptions.tsx

```diff
@@ -76,6 +76,14 @@ const buttonStuckCss = css`
 	}
 `;
 
+const fromAppHeadingCss = css`
+	${textSans.large({ fontWeight: 'bold' })}
+	color:${palette.brand[500]};
+	line-height: 1.35;
+	border: 0;
```

Comment:
Is this needed? There shouldn't be a border on the `h2` to start with or is it picking it up from somewhere else?

Reply:
Good catch, had this as a CSS override on <Heading> at first, but you can't remove the border from that